src/components/Tree/types.ts

```diff
@@ -0,0 +1,20 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ReactElement, ReactNode } from 'react';
+
+import { OnDropCallback } from '@components/DropZone';
+
+export type TreeProps = {
+    id: string;
+    draggable?: boolean;
+    children: ReactElement<TreeItemProps> | Array<ReactElement<TreeItemProps>>;
+};
+
+export type TreeItemProps = {
+    id: string;
+    sort: Nullable<number>;
+    onNodeSelect?: (id: Nullable<string>) => void;
+    onDrop?: OnDropCallback<{ id: string; sort: Nullable<number> }>;
+    component: ReactNode;
```

Comment:
Rename `component` property to `contentComponent` make it optional and create the default component to be used, where we pass the label to TreeItem.

Reply:
I'm discussing this with the Fondue team this afternoon